client/pages/about-us/profile-Card.js

```diff
@@ -0,0 +1,34 @@
+import ProfileComponent from "./profileCard-component.js";
```

Comment:
please, call this file profilecard-component.js

Fix path to ProfileComponent
`import ProfileComponent from "./profile-component.js";`